### ISSUE TITLE
Unable to build with 2.1.0 in Unity 2017.4.13f1

### ISSUE THREAD

**Issue by MaikelChan** [Unity-Technologies/PostProcessing#711]
##### What happened?

Unable to build project with version post-processing stack 2.1.0. I get the following error: `The type or namespace name `NUnit' could not be found. Are you missing an assembly reference?` in PostProcessingRuntimeTests.cs.

##### Post-processing stack version (v1, v2)?

v2.

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?

Unity 2017.4.13f1, Windows 10 Pro x64, Standalone Windows.

**Comment by TheShoes** [comment]
any luck need fix or work around asap

**Comment by MaikelChan** [comment]
Nah, I'm just using the last compatible version which is 2.0.16-preview.

**Comment by TheShoes** [comment]
I got it working by deleting the test folder in the postprocessing folder.

**Comment by MaikelChan** [comment]
Will this ever be officially fixed? After several versions of Post Processing and Unity 2017.4, still with the same issue. So much for LTS :/

**Comment by aramanamu** [comment]
> I got it working by deleting the test folder in the postprocessing folder.

Thanks, this helped. Couldn't find much out there on it. My problem was the runtime tests.
